snapcraft/tests/store/test_store_client.py

```diff
@@ -0,0 +1,138 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+from unittest import mock
+
+import requests
+
+from snapcraft import (
+    storeapi,
+    tests
+)
+from snapcraft.storeapi import constants
+from snapcraft.tests import fixture_setup
+
+
+class LoginTestCase(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.useFixture(fixture_setup.FakeStore())
+        self.client = storeapi.StoreClient()
+
+    def test_login_successful(self):
```

Comment:
yes, look down:
test_failed_login_with_wrong_one_time_password

That's only in unit tests though. We could add one for integration, even hitting the staging test. I'm not sure if it adds value, but it's easy to do. File a bug if you want it, and assign it to me.